cli/ts/commands/airdrop.ts

```diff
@@ -1,28 +1,21 @@
 import { BaseContract } from "ethers";
 import { type MACI, type TopupCredit, getDefaultSigner, parseArtifact } from "maci-contracts";
 
-import { banner } from "../utils/banner";
-import { contractExists } from "../utils/contracts";
-import { readContractAddress } from "../utils/storage";
-import { logError, logGreen, success } from "../utils/theme";
+import { type AirdropArgs, logError, logGreen, success, readContractAddress, contractExists, banner } from "../utils";
 
 /**
  * Utility that can be used to get
  * topup credits aidropped
  * to the coordinator
- * @param amount the amount of credits to airdrop
- * @param contractAddress the address of the ERC20 contract
- * @param pollId the id of the poll
- * @param maciAddress the address of the MACI contract
- * @param quiet whether to log the output
+ * @param AirdropArgs - The arguments for the airdrop command
```

Comment:
I think it should be like this to navigate between types. Not sure if current approach will work with typedoc and navigation.

```suggestion
 * @param {AirdropArgs} - The arguments for the airdrop command
```

Reply:
<img width="716" alt="Screenshot 2024-01-18 at 17 46 55" src="https://github.com/privacy-scaling-explorations/maci/assets/93448202/c7fd4e39-7f4f-4305-80b2-a5d011bf909a">

![Uploading Screenshot 2024-01-18 at 17.47.02.png…]()

Reply:
after typedoc generates, it links good

Reply:
is that ok?